man/make.conf.5

```diff
@@ -1208,6 +1208,15 @@ Defaults to /var/tmp.
 
 This should not be set to point anywhere under location of any repository.
 .TP
+\fBPORTAGE_TRUST_HELPER\fR = \fI[path]\fR
+Defines an executable file which initializes and maintains
```

Comment:
This says PORTAGE_TRUST_HELPER is an "executable file", but it actually seems to be a shell command line.

Reply:
Fixed